project_timelog/controllers/main.py

```diff
@@ -1,104 +1,83 @@
 # -*- coding: utf-8 -*-
 import datetime
-import time
-import openerp
 from openerp import http
 from openerp.http import request
-from openerp.addons.base import res
-import werkzeug
 
-class Controller(openerp.addons.bus.bus.Controller):
-    def _poll(self, dbname, channels, last, options):
-        if request.session.uid:
-            registry, cr, uid, context = request.registry, request.cr, request.session.uid, request.context
-            channels.append((request.db, 'project.timelog', request.uid))
-        return super(Controller, self)._poll(dbname, channels, last, options)
+
+class TimelogController(http.Controller):
 
     @http.route('/timelog/init', type="json", auth="public")
     def init_timelog(self, **kwargs):
-        current_user = request.env["res.users"].search([("id", "=", http.request.env.user.id)]) # current user
-        current_user_active_task_id = current_user.active_task_id # activ task for current user
-        current_user_active_work_id = current_user.active_work_id # activ subwork for current user
-
-        stopline = request.env["project.task"].search([('id', '=', current_user_active_task_id)]) # stopline for current task
-
-        # All logs
-        all_timelog =  request.env["project.timelog"]
+        timelog_obj = request.env["project.timelog"]
+        all_timelog_current_user = timelog_obj.search([("user_id", "=", http.request.env.user.id)])
```

Comment:
It's could be too heavy after one year of using timer.

Try to retrict it to one week at least